### ISSUE TITLE
[css-color-6] Behavior of color-contrast() when no level is specified 

### ISSUE THREAD

**Issue by LeaVerou** [w3c/csswg-drafts#7556]
[Right now](https://drafts.csswg.org/css-color-6/#colorcontrast), if no level is specified, the behavior is that "the most contrasting color" is returned. 
This essentially ignores the preference order the candidates are listed in and is vastly different from the behavior when a level is specified, breaking consistency. E.g. it makes `color-contrast(over teal, white, black)` the same as `color-contrast(over teal, black, white)`.
What are the use cases where authors want the most contrasting color? Why not default to a reasonable (or even highest) level?
